snap/remote/remote.go

```diff
@@ -39,6 +42,7 @@ type Snap struct {
 	Prices          map[string]float64 `json:"prices,omitempty"`
 	Publisher       string             `json:"publisher,omitempty"`
 	RatingsAverage  float64            `json:"ratings_average,omitempty"`
+	Revision        string             `json:"revision"`
```

Comment:
no, it's actually an integer:
curl https://search.apps.ubuntu.com/api/v1/package/hello-world.mvo

... "revision": 1, ...